fasttrack/cli.py

```diff
@@ -327,6 +363,16 @@ def _load_data_and_meta(
                 google_sheets["dataset_meta"], google_sheets["variables_meta"], google_sheets["sources_meta"]
             )
             data = sheets.parse_data_from_sheets(google_sheets["data"])
+        except urllib.error.HTTPError:
+            _bail(
+                [
+                    sheets.ValidationError(
+                        "Sheet not found, have you copied the template? Creating new Google Sheets document or new "
+                        "sheets with the same name in the existing document does not work."
+                    )
+                ]
+            )
+            continue
```

Comment:
It would be nice if we could add a URL-validator. So if the user inputs a URL that does not work in the first input field, an error message box is shown to them with sth like "The URL appears to be invalid. Please check!".

Reply:
That's a good idea, putting it on my list of improvements.